core/chains/evm/client/errors.go

```diff
@@ -465,3 +465,15 @@ func NewSendErrorReturnCode(err error, lggr logger.Logger, tx *types.Transaction
 	}
 	return clienttypes.Unknown, err
 }
+
+// SendOnlyErrorParse handles SendOnly nodes error codes. In that case, we don't assume there is another transaction that will be correctly
+// priced.
+func SendOnlyErrorParse(err error) clienttypes.SendTxReturnCode {
```

Comment:
`Parse` is typically a prefix in Go:
```suggestion
func ParseSendOnlyError(err error) clienttypes.SendTxReturnCode {
```
Although, is regex matching without pulling any data out really 'parsing'?

Reply:
Maybe `Classify` since it returns a code? Not sure, I've been struggling to find a good name for this, since the beginning.